grafana-plugin/src/pages/insights/Insights.hooks.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+
+import { useStore } from 'state/useStore';
+
+export const useAlertCreationChecker = () => {
+  const {
+    alertGroupStore: { updateAlertGroups, alerts },
+  } = useStore();
+  const [isFirstAlertCheckDone, setIsFirstAlertCheckDone] = useState(false);
+
+  const isAnyAlertCreatedMoreThan20SecsAgo = Array.from(alerts).some(([_key, alert]) => {
+    const alertTime = new Date(alert.started_at).getTime();
+    const nowTime = new Date().getTime();
+    return nowTime - alertTime > 20_000;
```

Comment:
might be a good idea to put these "magic numbers" in separate consts for better readability